templates/ingress.yaml

```diff
@@ -0,0 +1,76 @@
+---
+
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ getSiteName(True) }}
+  namespace: {{ data.namespace }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    {%- if data.nginxConfigurationSnippet %}
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      {{ data.nginxConfigurationSnippet | indent(6) }}
+    {%- endif %}
+
+spec:
+  {%- if not data.noTls and data.noTls is not defined %}
```

Comment:
Why do you need both `not` and `not defined`?

Reply:
No, you are right I don't need both